src/App.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import AddItem from './components/AddItem/AddItem.js';
```

Comment:
what do you to organize those imports  :D

first, the imports from other libraries like to react
them our components like this:

import React from 'react';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';

import './App.css';
import List from './components/List/List.js';
import NavBar from './components/NavBar/NavBar.js';

the reason is due to, when you will see the code, you can see each section of imports

isn't mandatory, just if you want  🕺

Reply:
Thank you for your feedback @FauricioValencia